### PR TITLE
New moth customization set

### PR DESCRIPTION
**Pull request by Mifutus**
## About The Pull Request

image
image

Added new mothwings, markings and antennae as seen above.

## Why It's Good For The Game

Everyone loves more moth variety!

## Changelog

:cl:
add: Added new moth customizations available in the character creation under the name "Dipped"
image: added icons for the moth wings, antennae and markings
:cl:

### CODE REVIEW

**Comment by Jackal-boop** [comment]
praying that you requested permission to make a species change from maints so you are not struck down by a bolt of lightning

**Comment by DaliIsTaken** [comment]
Really cool design! ^^

**Comment by Mifutus** [comment]
> Really cool design! ^^

Thank you!!

**Comment by Yobrocharlie** [comment]
ABYSS MOTH,
Amazing pr

**Comment by Jacquerel** [comment]
it seems like you accidentally added a single stray pixel to `m_moth_wings_feathery_FRONT`

**Comment by Mifutus** [comment]
> it seems like you accidentally added a single stray pixel to `m_moth_wings_feathery_FRONT`

Sorry about that! Should be fixed now, shouldn't work with tired eyes..

**Review by Wallemations — APPROVED** [review]
I don't play moths so I don't got a big opinion on 'em but the sprite looks fine to me so I don't see any reason not to

File: code/datums/sprite_accessories.dm [hunk 1]
```diff
@@ -2418,3 +2426,7 @@
 /datum/sprite_accessory/moth_markings/lightbearer
 	name = "Lightbearer"
 	icon_state = "lightbearer"
+
+/datum/sprite_accessory/moth_markings/dipped
+	name = "Dipped"
+	icon_state = "dipped"
```
Comment (Ghommie) [inline code/datums/sprite_accessories.dm:2432]:
```suggestion
	icon_state = "dipped"
	
```
end of file empty line

**Pull request merged**